maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+            else:
+                role = getattr(msg, "role", "user")
+                content = getattr(msg, "content", "")
+
+            parsed_messages.append(
+                GenerationRequestMessage(role=str(role), content=str(content))
+            )
+
+        return parsed_messages
+
+    @staticmethod
+    def get_model_params(**kwargs: Any) -> Dict[str, Any]:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)